global.R

```diff
@@ -6,6 +6,8 @@ library(shinythemes)
 library(markdown)
 library(ggplot2)
 library(plotly)
+library(mvtnorm)
+library(rpact)

```

Comment:
I added these two packages.

Reply:
Thanks, Dong. I had forgotten to add this to the file.